staging/src/k8s.io/client-go/tools/events/event_broadcaster.go

```diff
@@ -314,3 +318,45 @@ func (e *eventBroadcasterImpl) StartRecordingToSink(stopCh <-chan struct{}) {
 		stopWatcher()
 	}()
 }
+
+type eventClientImpl struct {
+	coreClient         typedv1core.EventsGetter
+	coreBroadcaster    record.EventBroadcaster
+	v1beta1Client      typedv1beta1.EventsGetter
+	v1beta1Broadcaster EventBroadcaster
+}
+
+// NewEventClient Creates a new event broadcaster.
```

Comment:
// NewEventClient creates a wrapper around new and legacy broadcasters to simplify
// migration of individual components to the new events api.

Reply:
Done.